rasa/cli/arguments/data.py

```diff
@@ -31,6 +31,12 @@ def set_convert_arguments(parser: argparse.ArgumentParser, data_type: Text) -> N
     )
 
     parser.add_argument("-l", "--language", default="en", help="Language of data.")
+    parser.add_argument(
```

Comment:
how about using the `filter` method to check if we can use the `Watson` training data importer instead of adding another CLI flag?

Reply:
@wochinge - I would still have to read the JSON of watson because within the metadata, there is an indication that it is indeed a watson exported file, how to differentiate between the regular Rasa json and watson json?

Reply:
I removed the argument and now i use the fitler method to detect watson data.